### PR TITLE
Improve code style

### PR DESCRIPTION
**Pull request by FooBarQuaxx**

### CODE REVIEW

File: src/sultan/api.py [hunk 1]
```diff
@@ -92,25 +90,16 @@ def load(cls, cwd=None, sudo=False, user=None, hostname=None, env=None, logging=
             context['user'] = 'root' if sudo else getpass.getuser()
         context.update(kwargs)
 
-        s = Sultan(context=context)
-        return s
+        return cls(context=context)
```
Comment (davydany) [inline src/sultan/api.py]: Today I learned something new! It makes perfect sense, but I have been writing this the wrong way!

**Comment by davydany** [comment]
Thank you @FooBarQuaxx! You're awesome!

File: src/sultan/api.py (hunk 1 above)
Comment (FooBarQuaxx) [inline src/sultan/api.py]: I don't think what you've done is wrong. It just feels more logical to use `cls`, the class object parameter, for instantiation.

Comment (davydany) [inline src/sultan/api.py]: I agree! I was basically thanking you for teaching me something new :-) :+1:

**Pull request merged**